cypress.json

```diff
@@ -1,5 +1,16 @@
 {
   "baseUrl": "http://localhost:3005",
   "viewportHeight": 800,
-  "viewportWidth": 1280
+  "viewportWidth": 1280,
+  "testFiles": [
+    "login.spec.js",
+    "modelPlan.spec.js",
+    "collaborator.spec.js",
+    "discussions.spec.js",
+    "documents.spec.js",
+    "characteristics.spec.js",
+    "participantsAndProviders.spec.js",
+    "opsEvalAndLearning.spec.js",
+    "archive.spec.js"
```

Comment:
Does this achieve what we're looking for? I ask because we're running `scripts/dev db:seed` on each test file, so we _should_ have a fresh database for each test!

https://github.com/CMSgov/mint-app/blob/0b35902a1f707477716706b5d7bb68428b26b4eb/cypress/support/global-hooks.js#L1-L3

Reply:
Hmmm.  The two test files following the spec where model plan was archived - they were consistently failing to find the 'Empty Plan'.  I'll look into this further as well as if it is indeed seeding on each test file

Reply:
https://github.com/CMSgov/mint-app/blob/1349485a8367358ba15469f458245c32ed6341e9/scripts/data/local-db.sql#L24

Hmm - it does appear to be in the data dump 🤔 perhaps because `db:seed` isn't wiping the database, any changes to that model plan (by ID) will not properly get reflected?

@patrickseguraoddball maybe try adding `cy.exec('scripts/dev db:clean')` to that `global-hooks` (before the seed) as well?